notebooks/cookbook/openmm_8_tutorial_rest_draft_v2.ipynb

```diff
@@ -0,0 +1,1080 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "73344755",
+   "metadata": {},
+   "source": [
+    "# Run a REST simulation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e110b4e9",
+   "metadata": {},
+   "source": [
+    "Replica exchange solute tempering (REST) is an enhanced sampling approach that aims to improve sampling for a user-defined subset of atoms (i.e., REST atoms) by increasing its effective temperature.\n",
+    "\n",
+    "REST involves running replica exchange to sample a set of thermodynamic states, where one state contains the unmodified Hamiltonian and all other states involve modified Hamiltonians where REST atom interactions are scaled to increased temperatures (Wang et al. 2011, DOI: 10.1021/jp204407d).\n",
+    "\n",
+    "In this tutorial, we show how to build an REST-capable OpenMM system. At the end, we show how to use OpenMMTools (https://github.com/choderalab/openmmtools) objects to run REST."
```

Comment:
It might be better to link to the documentation (https://openmmtools.readthedocs.io) rather than the github repo.